### PR TITLE
Remove reference to Krux in pressed pages

### PR DESCRIPTION
**Pull request by GHaberis**
## What does this change?

Remove final references to Krux from pressed page templates, following main clean up: https://github.com/guardian/frontend/pull/22341

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-pixel.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// @flow
+import config from 'lib/config';
+
+export const twitterPixel: () => ThirdPartyTag = () => ({
+    shouldRun: config.get('switches.twitterTrackingPixel', false),
+    url: '',
+    useImage: true,
+});
```
Comment (buck06191) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/twitter-pixel.js]: Why is this added in here? Is this something that was part of Krux and needed keeping?

**Review by buck06191 — APPROVED** [review]
Looks good to me apart from the Twitter pixel code. If this is intentional, can we add something to this PR about why this is being added in as part of the Krux removal?

**Comment by GHaberis** [comment]
> Looks good to me apart from the Twitter pixel code. If this is intentional, can we add something to this PR about why this is being added in as part of the Krux removal?

Not intentional at all! I accidentally committed this file, tried to delete but Github was down/slow so it took a while to show my commit! It's gone now.
